java/plasma/src/main/java/org/apache/arrow/plasma/PlasmaClientJNI.java

```diff
@@ -46,11 +46,6 @@ public static native ByteBuffer create(long conn, byte[] objectId, int size, byt
 
   public static native boolean contains(long conn, byte[] objectId);
 
-  public static native void fetch(long conn, byte[][] objectIds);
```

Comment:
Considering the backward compatibility, we should keep those APIs and mark as ```deprecated```. Thoughts?

Reply:
These two APIs have native declare, but implementation has been removed.

Reply:
So it's not for this JIRA? Better file a separated PR addressing it.